### PR TITLE
Fix: Make the service fail early if environment variables are not defined.

### PR DESCRIPTION
**Pull request by redaphid**
Would anyone mind if we made the service explode if it doesn't have the environment variables it needs to run? 
This would make it easier to debug local dev, and give us some safeguard when deploying.

### CODE REVIEW

File: packages/api/src/constants.js [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+  SALT,
+  SENTRY_DSN,
+  DATABASE_TOKEN,
+  CLUSTER_SERVICE,
+  LOGTAIL_TOKEN,
+  MAILCHIMP_API_KEY,
+  CLUSTER_API_URL,
+  DATABASE_URL,
+  DEBUG,
+} = process.env
```
Comment (alanshaw) [inline packages/api/src/constants.js:17]: This is not a node application, it's a service worker. There's no process object and no access to the filesystem so this won't work.

File: packages/api/src/constants.js [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+
+if (!SALT) {
```
Comment (alanshaw) [inline packages/api/src/constants.js:19]: Accessing any of these globals when they are undefined will throw an error so we already fail fast.

This is also why you see `typeof CLUSTER_BASIC_AUTH_TOKEN !== 'undefined'` in this file for example, because this var is optional.

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/api/src/constants.js (hunk 1 above)
Comment (redaphid) [reply packages/api/src/constants.js:17]: Alright, no worries. Thanks for taking the time to review it.

File: packages/api/src/constants.js (hunk 2 above)
Comment (redaphid) [reply packages/api/src/constants.js:19]: Is this new(ish)? I ended up doing this because I was missing env vars, and had to dig around trying to find what was missing.

That said, we'd have to go a different route if we're not using process.env. 
I'll see if I hit it again, and open a new PR if that happens.

**Comment by redaphid** [comment]
tl;dr it's not a node app. Thanks for taking the time to review. I'll document where the undefined vars were messing me up if I run into them again.
